test/Services/Blamable/HostnetBlamableProviderTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+namespace Hostnet\Bundle\EntityTrackerBundle\Services\Blamable;
+
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+
+/**
+ * @covers Hostnet\Bundle\EntityTrackerBundle\Services\Blamable\HostnetBlamableProvider
+ * @author Eunice Valdez <evaldez@hostnet.nl>
+ */
+class HostnetBlamableProviderTest extends \PHPUnit_Framework_TestCase
+{
+    private $token_storage;
+
+    protected function setUp()
+    {
+        $this->token_storage = $this->getMock(TokenStorageInterface::class);
```

Comment:
Please change this to the FQNS in a string. We're also supporting php and the travis check for that version broke.

```
$this->getMock('Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface');
``